regression/pages/studio/grading_studio.py

```diff
@@ -0,0 +1,56 @@
+"""
+Grading Page for Studio
+"""
+import urllib
+
+from edxapp_acceptance.pages.studio.settings_graders import GradingPage
+
+from regression.pages.studio.utils import press_the_notification_button
+from regression.pages.studio.utils import get_course_key
+from regression.pages.studio import BASE_URL
+
+
+class GradingPageExtended(GradingPage):
+    """
+    Grading Page for Studio
+    """
+    @property
+    def url(self):
+        """
+        Construct a URL to the page within the course.
+        """
+        course_key = get_course_key(self.course_info)
+        url = "/".join(
+            [BASE_URL, self.url_path, urllib.quote_plus(unicode(course_key))])
+        return url if url[-1] is '/' else url + '/'
+
+    def letter_grade(self, selector):
+        """
+        Returns: first letter of grades on grading page
```

Comment:
Could you give an example? When I hear this, it sounds like a grading page has multiple grades, and each grade has multiple _more_ grades. I'm guessing that I misunderstand, so maybe you could illustrate or clarify?